README.md

```diff
@@ -99,7 +99,7 @@ For detailed installation instructions, see the
 -   **Operating system**: macOS / OS X · Linux · Windows (Cygwin, MinGW, Visual
     Studio)
 -   **Python version**: Python 2.7, 3.5+ (only 64 bit)
--   **Package managers**: [pip] · [conda] (via `conda-forge`)
```

Comment:
Sorry if this looked confusing, but this is intentional. The "via conda-forge" in brackets is an additional note, not the link. The link gets provided via the `[conda]` reference below:

```markdown
[conda]: https://anaconda.org/conda-forge/spacy
```